views/partials/summary.ejs

```diff
@@ -13,7 +13,7 @@
 	<span>
 		with a
 		<%= Math.round(data.windSpeed) %>
-		<span><%= (params.units === 'si') ? 'm/s' :'mph' %>
+		<span><%= params.windUnit %>
 		<% if (params.units === 'si' && data.windSpeed < 13.9 || data.windSpeed < 32) { %>
```

Comment:
These are screenshots from the same location/same time after switching units:

![cmph](https://cloud.githubusercontent.com/assets/22562624/19222831/e6a0b928-8e61-11e6-9a32-c251cc2ca633.png)
![fmph](https://cloud.githubusercontent.com/assets/22562624/19222832/e6b9b1c6-8e61-11e6-85dc-0051921a940a.png)
![kmh](https://cloud.githubusercontent.com/assets/22562624/19222833/e6c234fe-8e61-11e6-95ae-04e27efa1996.png)
![ms](https://cloud.githubusercontent.com/assets/22562624/19222834/e6c9ca84-8e61-11e6-9a66-01ff9fd5fb95.png)